website/docs/docs/develop/run-your-dbt-projects.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Run your dbt projects"
+id: "run-your-dbt-projects"
+---
+You can run your dbt projects with [dbt Cloud](/docs/develop/dbt-cloud-features) and [dbt Core](https://github.com/dbt-labs/dbt-core). dbt Cloud is a hosted application where you can develop directly from a web browser. dbt Core is an open source project where you can develop from the command line.
+
+Among other features, dbt Cloud provides a development environment to help you build, test, run, and [version control](/docs/collaborate/git-version-control) your project faster. It also includes an easier way to share your [dbt project's documentation](/docs/collaborate/build-and-view-your-docs) with your team. These development tasks are directly built into dbt Cloud for an _integrated development environment_ (IDE). Refer to [Develop in the Cloud](/docs/develop/develop-in-the-cloud) for more details.
+
+With dbt Core, you can run your dbt projects from the command line. The command line interface (CLI) is available from your computer's terminal application such as Terminal and iTerm. When using the command line, you can run commands and do other work from the current working directory on your computer. Before running the dbt project from the command line, make sure you are working in your dbt project directory. Learning terminal commands such as `cd` (change directory), `ls` (list directory contents), and `pwd` (present working directory) can help you navigate the directory structure on your system.
+
+When you run your dbt project, the commands you commonly use are:
```

Comment:
This felt a little Core focused, but it's true for either product. Maybe a tiny tweak will clarify
```suggestion
When running your dbt project from dbt Core or dbt Cloud, you'll commonly use:
```

Reply:
```suggestion
When running your project from dbt Core or dbt Cloud, the commands you commonly use are:
```